packages/core/content-type-builder/admin/src/components/ContentTypeBuilderNav/ContentTypeBuilderNav.tsx

```diff
@@ -63,7 +62,9 @@ export const ContentTypeBuilderNav = () => {
                       {link.links.map((subLink: any) => (
                         <SubNavLink
                           as={NavLink}
-                          // @ts-ignore
+                          // verify if "to" is needed
+                          // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+                          // @ts-expect-error
```

Comment:
@Bassel17 did you read the feedback it gives you? if you do `// @ts-expect-error MY COMMENT` eslint won't complain :)